typings/src/ui/store.ts

```diff
@@ -5,12 +5,12 @@ type Login = {
   checkRegistered?: boolean;
 };
 
-export interface IGetUserInfo {
+export type GetUserInfo = {
   profileDid: string;
```

Comment:
I have thought of doing the same change and I figured it will be easier to do it in a separate PR because we have the naming of `profileDid` outside this PR as well. I didn't want to name it `profileDid` in this PR but did it anyway to stay consistent with what we have elsewhere and make the correction on a separate PR.